codec/console/dec/src/h264dec.cpp

```diff
@@ -505,14 +516,27 @@ int32_t main (int32_t iArgC, char* pArgV[]) {
   sDecParam.sVideoProperty.size = sizeof (sDecParam.sVideoProperty);
   sDecParam.eEcActiveIdc = ERROR_CON_SLICE_MV_COPY_CROSS_IDR_FREEZE_RES_CHANGE;
 
+#if defined(NODEFS)
+  EM_ASM(
+    FS.mkdir('/workload');
+    FS.mount(NODEFS, { root: '.' }, '/workload');
+  );
+#endif
+
   if (iArgC < 2) {
     printf ("usage 1: h264dec.exe welsdec.cfg\n");
     printf ("usage 2: h264dec.exe welsdec.264 out.yuv\n");
     printf ("usage 3: h264dec.exe welsdec.264\n");
     return 1;
   } else if (iArgC == 2) {
     if (strstr (pArgV[1], ".cfg")) { // read config file //confirmed_safe_unsafe_usage
-      CReadConfig cReadCfg (pArgV[1]);
+      string cfgFilePath;
+#if defined(NODEFS) || defined(MEMFS)
+      cfgFilePath = string(CWD) + string(pArgV[1]);
```

Comment:
Why is the `string(CWD) + ` needed?

Reply:
This is for the [Emscripten File System](https://emscripten.org/docs/porting/files/index.html)

node.js environment for example, we map the main project dir to `/workload` in virual filesystem. 

After the mount we can run h264enc.js like: `node h264enc.js testbin/xxx.cfg`

The `string(CWD) +` add the path prefix. Without it we may need to run  h264enc.js like `node h264enc.js workload/testbin/xxx.cfg`

Reply:
Why map the main project dir to `/workload` instead of just the `/`?

Reply:
Yes, I tried to mount host dir `./` to `/` in virtual FS for nods.js env, but this will cause error. 

In browser env, packaging all the main will lead to longer file reading time.